packages/server/modules/auth/defaultApps.js

```diff
@@ -103,7 +103,8 @@ const SpeckleDesktopApp = {
     'streams:write',
     'profile:read',
     'profile:email',
-    'users:read'
+    'users:read',
+    'users:invite'
```

Comment:
Would be great if you'd add the new scopes to `/packages/server/modules/core/helpers/mainConstants.js` and use those constants here, instead of duplicating the strings